src/api/authApi.js

```diff
@@ -0,0 +1,57 @@
+import config from "../config/config";
+
+export const getAuthUser = async () => {
+  const request = {
+    method: "GET",
+    credentials: "include",
+  };
+
+  try {
+    const response = await fetch(
+      `${config.REACT_APP_REQUEST_URL}/auth/users`,
+      request
+    );
+    const data = response.json();
+
+    return data;
+  } catch (error) {
+    return console.log(error);
+  }
+};
+
+export const findOrCreateUser = async (userData, token) => {
```

Comment:
```suggestion
export const fetchRegisteUser = async (userData, token) => {
```
적절한 함수명이 뭐가 있을까요??
여기서 유저를 찾거나 만드는 행위는 하지않아서 네이밍이 부자연스러워 보입니다

Reply:
음 고민해보니 `fetch`라는 단어가 들어가면 추후 `fetch`관련 유틸함수나 커스텀 훅이 제작되었을때 헷갈릴 수도 있을 것 같아서, 여기서 `getUser`로 하고, 이미 세션쿠키가 발급되어 인증된 사용자의 정보를 가져오는 함수는 `getAuthUser`로 하면 어떨까요??